### ISSUE TITLE
Does not calculate correctly with some 0.17 science

### ISSUE THREAD

**Issue by zearmhot** [KirkMcDonald/kirkmcdonald.github.io#134]
Some science packs now yield multiple per craft. The calculator does not factor this in.

For example, you get 3x utility science packs per craft. So I put in that I want "60 items/min". That means only 20 crafts per minute. However, it assumes I want 60 crafts per minute and claims I need (40!) assembly machines making low density parts.

I vastly overbuilt my factory before realizing it was doing this. I assume this is not intended? I would at least reword it to "crafts/minute" if this is intended.

**Comment by adrianvanderwal** [comment]
![image](https://user-images.githubusercontent.com/6479661/54100675-ade71b80-43fa-11e9-85d2-32f2fed589dd.png)

In your example of LDS, I think you are forgetting the time factor (20 second craft times).

In my example maths, I'm using Assembler 1's
Simple maths means:
60 Utility Packs = 60 LDS per minute
1 LDS takes 1/(0.5/20) = 40 seconds to make
60 LDS takes 40*60 = 2400 seconds to make, which equates to 40 assemblers to make 60 per minute.

The calculator is not wrong.

**Comment by KirkMcDonald** [comment]
Yep. Those 20 second craft times mean you need a lot of assembling machines. The calculator is, indeed, reporting items/minute correctly.
